test/e2e/specs/adding-blocks.test.js

```diff
@@ -50,6 +50,7 @@ describe( 'adding blocks', () => {
 
 		// Using the regular inserter
 		await page.click( '.edit-post-header [aria-label="Add block"]' );
+		await page.waitForSelector( '.editor-inserter__menu' );
```

Comment:
I'm pretty sure that's not the only place in our tests where we open the inserter, we should probably be consistent.
Do you think we should factorize that in a `insertBlock( 'core' )` utility?
And also, can we add a comment explaining why this is necessary? Is there an API request or something?